src/utils.py

```diff
@@ -623,6 +634,30 @@ def get_min_age(beneficiary_dtls):
     min_age = min(age_list)
     return min_age
 
+def readOTP():
+    """
+    This function reads latest OTP from imessages based on handle_id being 19.
+    :return: OTP
+    """
+    conn = sqlite3.connect("/Users/vidyanarayanan/Library/Messages/chat.db")
```

Comment:
WE should not hardcode this path. Do you think its a good idea to replace `/Users/vidyanarayanan/` with `~/`? This will ensure that it works for everyone.